timer/api/store_test.go

```diff
@@ -240,7 +240,7 @@ func TestTimerUpdate(t *testing.T) {
 	require.True(t, errors.ErrorEqual(err, ErrEventIDNotMatch))
 	require.Equal(t, tpl, *tm)
 
-	// test apply without check
+	// test apply without check1
```

Comment:
Is this a typo?

Reply:
No, there is check2, check3 later. But I'll update the comment message to make it easier to read